### ISSUE TITLE
[Feature Request]  ability to access counts of active matches and invitations in turn based multiplayer 

### ISSUE THREAD

**Issue by n8stowell82** [playgameservices/play-games-plugin-for-unity#396]
It would be really nice to be able to call out and get a list of current active matches and pending invitations. At the very least counts for both.  This way the user can always see an in came match count as well as invitation count. An example would be a button that shows how many matches the user has that are waiting on them to take their turn.

**Comment by spotavi** [comment]
This is a nice feature to have. :+1: 

Also is there an ETA for all the things broken/fixes with the dodge release ...

**Comment by ZenithCode** [comment]
+1 for ETA on all broken/ fixes with dodge.

**Comment by claywilkinson** [comment]
Closing this as a duplicate of #256
